src/ds/champ_map.h

```diff
@@ -428,13 +428,13 @@ namespace champ
     Map<K, V, H> map;
     CBuffer serialized_buffer;
 
-    struct pair
+    struct KVTuple
```

Comment:
That's still a pair of data elements, essentially, despite the presence of the hashing function?

Reply:
Yeah, not too sure about the name here. I think `KVTuple` is generic enough and it's only used (so far) by one function anyway. Happy to change it though.